apps/heft/src/plugins/SassTypingsPlugin/SassTypingsGenerator.ts

```diff
@@ -0,0 +1,96 @@
+import * as path from 'path';
+import { render, Result } from 'node-sass';
+import * as postcss from 'postcss';
+import * as cssModules from 'postcss-modules';
+import { StringValuesTypingsGenerator, IStringValueTypings } from '@rushstack/typings-generator';
+import { LegacyAdapters } from '@rushstack/node-core-library';
+
+/**
+ * @public
+ */
+export interface ISassTypingsGeneratorOptions {
+  buildFolder: string;
+}
+
+/**
+ * @internal
+ */
+interface IClassMap {
+  [className: string]: string;
+}
+
+/**
+ * Generates type files (.d.ts) for Sass/SCSS/CSS files.
+ *
+ * @public
+ */
+export class SassTypingsGenerator extends StringValuesTypingsGenerator {
+  private _classMap: IClassMap;
```

Comment:
this can be made local to _getClassNamesFromCSSAsync it seems?

Reply:
Good point! It is now local to the method.